File: Mistral Large/C++/2-3.c
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    ifstream inputFile("input.txt");
    ofstream outputFile("output.txt");

    if (!inputFile) {
        cout << "Error opening input file." << endl;
        return 1;
    }

    if (!outputFile) {
        cout << "Error opening output file." << endl;
        return 1;
    }

    string line;
    while (getline(inputFile, line)) {
        outputFile << line << endl;
    }

    inputFile.close();
    outputFile.close();

    cout << "File content copied successfully." << endl;
    return 0;
}

File: Mistral Large/Python/2-3.py
def file_copy():
    try:
        with open('input.txt', 'r') as input_file:
            content = input_file.read()

        with open('output.txt', 'w') as output_file:
            output_file.write(content)

        print("File content copied successfully.")
    except IOError as e:
        print("Error:", e)

file_copy()

